UFS-DA/ush/build_UFSDA.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+# build_UFSDA.sh
+# 1 - determine if on supported host
+# 2 - load modules
+# 3 - run ecbuild
+# 4 - build
+# 5 - optional, run unit tests
+
+set -ex
+
+cd ..
+pwd=$(pwd)
+
+run_tests=${1:-"NO"}
+dir_root=${2:-$pwd}
```

Comment:
should not need to do this

Reply:
Do you think tests should always run or should they never run?